demo/features/step_definitions/cookie_banner_test_steps.rb

```diff
@@ -5,24 +5,29 @@
 end
 
 Then("I can see a link to the cookie preferences page for England") do
-  expect(page).to have_link("change your cookie settings", href: "/cookie-preferences")
+  expect(page).to have_link("change your cookie settings",
+                            href: "/cookie-preferences/?cookie_prefs_return_url=http%3A%2F%2Flocalhost%3A3000%2Fshow-page")
```

Comment:
Reminder: need to check if we need to have other domain (i.e not localhost) in the tests. We have [domain_helper](https://github.com/citizensadvice/cookie-preferences/blob/771146f722b70c6b4c9d796be03c52912c1e17b4/demo/features/step_definitions/cucumber_cookie_management_test_steps.rb#L60) used for other tests but I'm not really sure in which cases it wouldn't be localhost in the pipeline

Reply:
I think checking if we're in `Rails.env.production?` is a good suggestion and would work well. However, the problem I'm having at the moment is that the Rails env is set for production when starting the dev server: https://github.com/citizensadvice/cookie-preferences/blob/main/.github/actions/start-demo-dev-server/action.yml#L17. I believe this is intentional so we get as close to the prod app as possible.